lib/src/base-styles/_nx-typography.scss

```diff
@@ -79,7 +79,6 @@
 }
 
 .nx-p {
-  font-size: $nx-font-size-s;
```

Comment:
I think there's actually a separate ticket for this: RSC-164

Reply:
> I think there's actually a separate ticket for this: RSC-164

https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/b087009955d6ea51d78283f511bc3ef303a686a8